azure-communication-ui/chat/build.gradle

```diff
@@ -47,13 +47,24 @@ dependencies {
     implementation "androidx.fragment:fragment-ktx:$androidx_fragment_ktx_version"
     implementation "androidx.navigation:navigation-fragment-ktx:$androidx_navigation_fragment_ktx_version"
 
-    implementation "com.microsoft.fluentui:fluentui_others:$microsoft_fluent_ui_version_v2"
-    implementation "androidx.activity:activity-compose:$androidx_activity_compose_version"
-
     api ("com.azure.android:azure-communication-chat:$azure_chat_sdk_version")
     api ("com.azure.android:azure-communication-common:$azure_common_sdk_version")
 
+    implementation "com.microsoft.fluentui:fluentui_others:$microsoft_fluent_ui_version_v2"
+    implementation "androidx.compose.ui:ui:$compose_version"
+    implementation "androidx.compose.material:material:$compose_version"
+    implementation "androidx.compose.ui:ui-tooling-preview:$compose_version"
```

Comment:
do we need to add material depepndency? I think we can avoid as per discussion with Adam

Reply:
Removing the material in next commit. Thanks